kratos/utilities/os_utilities.cpp

```diff
@@ -41,5 +43,82 @@ std::string GetCurrentWorkingDir()
     KRATOS_CATCH("")
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
+void Remove(const std::string& rFileName)
```

Comment:
do you want this to remove a file or a folder?

=> maybe separate fcts are better here?
`RemoveFile` & `RemoveDir`

Reply:
ok